docs/mindsdb-docs/docs/sql/api/insert.md

```diff
@@ -1,17 +1,22 @@
-# INSERT INTO Statement
+# `#!sql INSERT INTO` Statement
 
 ## Description
 
-The `#!sql INSERT INTO` statement is used to insert data into a table. This inserted data comes from the subselect statement. It is commonly used to input predictions into the database.
+The `#!sql INSERT INTO` statement inserts data into a table. The data comes from a subselect query. It is commonly used to input prediction results into a database table.
 
 ## Syntax
 
+Here is the syntax:
+
 ```sql
 INSERT INTO [integration_name].[table_name]
     (SELECT ...);
 ```
 
-It performs a subselect `#!sql (SELECT ...)` and gets data from there. Then, this data is inserted into a table of integration `#!sql [integration_name]` using `#!sql INSERT INTO TABLE [table_name]` statement.
+And the steps followed by the syntax:
+
+- It executes a subselect query to get the output dataset.
+- It uses the `INSERT INTO` statement to insert the output of the `(SELECT ...)` query into the `[integration_name].[table_name]`.
```

Comment:
We could add: 
Destination table must already content all columns that will be used for insert

Reply:
Added.